src/components/Tabs/TabsBar/__tests__/TabsBar.spec.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { render, RenderResult, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+import { TabsBar } from '../TabsBar';
+import { TabItem } from '../../TabItem';
+
+describe('<TabsBar>', () => {
```

Comment:
In other places we use `describe('TabsBar component', () => { ... }` naming style. 

Let's keep it consistent.